### PR TITLE
Remove v2 transition commands since everything is now v2

### PR DESCRIPTION
**Pull request by GregHanson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -603,48 +599,24 @@ isti%.yaml: $(HELM)
...
-# This is temporary. REMOVE ME after Envoy v2 transition
-# creates istio.yaml using values-envoyv2-transition.yaml
-generate_yaml-envoyv2_transition_loadbalancer_ingressgateway: $(HELM)
-	./install/updateVersion.sh -a ${HUB},${TAG} >/dev/null 2>&1
-	cat install/kubernetes/namespace.yaml > install/kubernetes/istio.yaml
-	$(HELM) template --set global.tag=${TAG} \
-		--namespace=istio-system \
-		--set global.hub=${HUB} \
-		--values install/kubernetes/helm/istio/values-envoyv2-transition.yaml \
-		--set gateways.istio-ingressgateway.type=LoadBalancer \
```
Comment (rshriram) [inline Makefile]: isn't this needed in the generate_yaml above?

Comment (rshriram) [reply Makefile]: ignore.. its set by default to load balancer..

**Comment by rshriram** [comment]
@GregHanson ready to push or need more changes?

**Comment by GregHanson** [comment]
@rshriram good to merge

**Pull request merged**